src/App.jsx

```diff
@@ -1,27 +1,33 @@
 import React from 'react';
```

Comment:
For the import order :

```suggestion
import React from 'react';
import { Switch, Route } from 'react-router-dom';

import NavBarr from './components/NavBarr';
import BienManger from './components/BienManger';
import Sport from './components/Sport';
import BienDansSaTete from './components/BienDansSaTete';
import Footer from './components/Footer';
import Home from './components/Home';

import './App.css';
```